### PR TITLE
Improve training script

### PR DESCRIPTION
**Pull request by djalova**
Addresses #2 and #8

### CODE REVIEW

File: train_eval.py [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+    batch_size:  Batch size dimension of returned tensor.
+    n:  number of entries per batch.
+
+  Returns:
+    A [batch_size, n] tensor of "boolean" numbers, where each number is
+    preresented as -1 or 1.
+  """
+
+  as_int = tf.random_uniform(
+      [batch_size, n], minval=0, maxval=2, dtype=tf.int32)
```
Comment (feihugis) [inline train_eval.py:88]: Would `maxval` be `1`?

**Review by feihugis — COMMENTED** [review]
LGTM for the bucket name issue #2. Leave a small comment about `train_eval.py`.

**Comment by lresende** [comment]
@feihugis @djalova What I was thinking here, was that, instead of requesting the user to provide a bucket name, we would use the current configured bucket name and append a suffix like a short UNID. Now, with the introduction with --clean, this would generate a new bucket every time --clean is used. Anyway, just my 0.02c.

**Comment by MLnick** [comment]
Closing

**Pull request closed without merging**